### PR TITLE
docs: translate integrate-in-editor page into Japanese

### PR DESCRIPTION
**Pull request by Spice-Z**
## Summary

Partial work of #880.
Sorry to late !

## Others

I found the typo in original document while translating.
I made a PR to fix it.
https://github.com/biomejs/biome/pull/1026

### CODE REVIEW

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+## 自分でプラグインを書く
+
+Biome は [LSP](https://microsoft.github.io/language-server-protocol/) を第一級でサポートしています。
+エディタが LSP を実装している場合、Biome の統合をシームレスに行うことができます。
+
+### LSP プロキシを使う
+
+Biome には `lsp-proxy` というコマンドがあります。 実行すると、Biome は2つのプロセスを生成します:
+- 要求された操作を実行する [デーモン](/ja/internals/architecture#デーモン-wip)
```
Comment (Spice-Z) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:65]: I'm not sure what url should be linked here.

In original document, the link is `https://biomejs.dev/internals/architecture/#daemon`.
But `#daemon` doesn't exist on that page.

I changed the link's tag to currently exists one.

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+標準入力を介して以下のコマンドを呼び出すことができます:
+- [`format`](/reference/cli/#biome-format)
+- [`lint`](/reference/cli/#biome-lint)
+- [`check`](/reference/cli/#biome-check)
+
+Biome は、新しい出力（変更が発生していない場合は元の出力）を [標準出力](https://ja.wikipedia.org/wiki/%E6%A8%99%E6%BA%96%E3%82%B9%E3%83%88%E3%83%AA%E3%83%BC%E3%83%A0#%E6%A8%99%E6%BA%96%E5%87%BA%E5%8A%9B_(stdout)) に返し、診断を [標準エラー](https://ja.wikipedia.org/wiki/%E6%A8%99%E6%BA%96%E3%82%B9%E3%83%88%E3%83%AA%E3%83%BC%E3%83%A0#%E6%A8%99%E6%BA%96%E3%82%A8%E3%83%A9%E3%83%BC%E5%87%BA%E5%8A%9B_(stderr)) に返します。
+
+`stdin` を使用する場合は、 `--stdin-file-path` オプションを渡す必要があります。
+ファイルの `path` は、ファイルシステムに存在する必要はなく、任意の名前で構いません。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:85]:
```suggestion
オプションに渡すファイルの `path` は、ファイルシステムに存在する必要はなく、任意の名前で構いません。 
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+- クライアント（エディタ）とサーバー（デーモン）の間のリクエストをプロキシするサーバー
+
+エディタがサーバーとやり取りできて [JSON-RPC](https://www.jsonrpc.org/) リクエストを送信できる場合、エディタがコマンドを実行するように設定するだけで済みます。
+
+[`neo-vim biome プラグイン`](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/server_configurations/biome.lua) がどのようにその処理を行っているか確認してみてください。
+
+
+### `stdin` を使う
+
+エディタが LSP をサポートしていない場合は、 [標準入力](https://ja.wikipedia.org/wiki/%E6%A8%99%E6%BA%96%E3%82%B9%E3%83%88%E3%83%AA%E3%83%BC%E3%83%A0#%E6%A8%99%E6%BA%96%E5%85%A5%E5%8A%9B_(stdin)) を利用して `biome` バイナリを直接使用します。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:75]:
```suggestion
エディタが LSP をサポートしていない場合は、 [標準入力](https://ja.wikipedia.org/wiki/%E6%A8%99%E6%BA%96%E3%82%B9%E3%83%88%E3%83%AA%E3%83%BC%E3%83%A0#%E6%A8%99%E6%BA%96%E5%85%A5%E5%8A%9B_(stdin)) を利用して `biome` バイナリを直接操作します。
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+標準入力を介して以下のコマンドを呼び出すことができます:
+- [`format`](/reference/cli/#biome-format)
+- [`lint`](/reference/cli/#biome-lint)
+- [`check`](/reference/cli/#biome-check)
+
+Biome は、新しい出力（変更が発生していない場合は元の出力）を [標準出力](https://ja.wikipedia.org/wiki/%E6%A8%99%E6%BA%96%E3%82%B9%E3%83%88%E3%83%AA%E3%83%BC%E3%83%A0#%E6%A8%99%E6%BA%96%E5%87%BA%E5%8A%9B_(stdout)) に返し、診断を [標準エラー](https://ja.wikipedia.org/wiki/%E6%A8%99%E6%BA%96%E3%82%B9%E3%83%88%E3%83%AA%E3%83%BC%E3%83%A0#%E6%A8%99%E6%BA%96%E3%82%A8%E3%83%A9%E3%83%BC%E5%87%BA%E5%8A%9B_(stderr)) に返します。
+
+`stdin` を使用する場合は、 `--stdin-file-path` オプションを渡す必要があります。
+ファイルの `path` は、ファイルシステムに存在する必要はなく、任意の名前で構いません。 
+**重要**なのは、Biome が**どのようにファイルを扱うべきか**判断するために、正しいファイル拡張子を指定することです。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:86]:
```suggestion
重要なのは、Biome が**どのようにファイルを扱うべきか**判断するために、正しいファイルの拡張子を指定することです。
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+1. releases タブからプラグインの .zip をダウンロードする。
+2. `⌘Сmd,` を押して IDE の設定を開き、**Plugins** を選択する。
+3. プラグインページで、**Settings** ボタンをクリックし、**Install Plugin from Disk...** をクリックする。
+
+## サードパーティのプラグイン
+
+これらは、各種エディタで使用できる、コミュニティによってメンテナンスされているプラグインです:
+- [`neovim`](https://neovim.io/): [`nvim-lspconfig`](https://github.com/neovim/nvim-lspconfig/) をインストールして、[instructions](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#biome) に従ってください
+- [`helix`](https://helix-editor.com/): [マニュアル](https://github.com/biomejs/biome/blob/main/editors/helix/manual.md) の手順に従ってください
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:50]:
```suggestion
- [`helix`](https://helix-editor.com/): [マニュアルの手順](https://github.com/biomejs/biome/blob/main/editors/helix/manual.md) に従ってください
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+- [`coc-biome`](https://github.com/fannheyward/coc-biome): [`coc.nvim`](https://github.com/neoclide/coc.nvim) 向けの Biome 拡張機能
+
+:::note
+ここにのっていないエディタのプラグインがありますか？ぜひ PR を送って、ここに追加してください！
+:::
+
+## 自分でプラグインを書く
+
+Biome は [LSP](https://microsoft.github.io/language-server-protocol/) を第一級でサポートしています。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:59]:
```suggestion
Biome は [LSP](https://microsoft.github.io/language-server-protocol/) を優先的にサポートしています。
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+
+- `@biomejs/cli-darwin-arm64`
+- `@biomejs/cli-darwin-x64`
+- `@biomejs/cli-linux-arm64`
+- `@biomejs/cli-linux-x64`
+- `@biomejs/cli-win32-arm64`
+- `@biomejs/cli-win32-x64`
+
+バイナリの名前は `biome` または `biome.exe` で、ライブラリのルートディレクトリにあります。 例: `@biomejs/cli-darwin-arm64/biome`、`@biomejs/cli-win32-x64/biome.exe`。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:99]:
```suggestion
バイナリの名前は `biome` または `biome.exe` で、ライブラリのルートディレクトリにあります。 例: `@biomejs/cli-darwin-arm64/biome`, `@biomejs/cli-win32-x64/biome.exe`
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+
+### バイナリを使ったデーモン
+
+CLI でバイナリを使うのは効率的ですが、ユーザーに[ログ](#デーモンのログ)を提供することができません。
+CLI からデーモンをブートストラップすることが可能で、その後はデーモンを介して CLI コマンドを使用することができます。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:104]:
```suggestion
CLI からデーモンを起動することが可能で、その後はデーモンを介して CLI コマンドを使用することができます。
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+
+そうしたい場合は、まず [`start`](/ja/reference/cli#biome-start) コマンドでデーモンプロセスを起動する必要があります:
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:106]:
```suggestion
この方法を試す場合は、まず [`start`](/ja/reference/cli#biome-start) コマンドでデーモンプロセスを起動する必要があります:
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+```
+
+その後、すべてのコマンドに `--use-server` オプションを追加する必要があります。 例:
+
+```shell
+echo "console.log('')" | biome format --use-server --stdin-file-path=dummy.js
+```
+
+:::note
+デーモンを使用する場合は、ゴーストプロセスを避けるために、[`stop`](/ja/reference/cli#biome-stop) コマンドでプロセスを再起動/終了する責任があります。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:120]:
```suggestion
デーモンを使用する場合は、プロセスが生き続けてしまうことを避けるために、[`stop`](/ja/reference/cli#biome-stop) コマンドでプロセスを再起動/終了する責任があります。
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+:::
+
+:::caution
+デーモン経由の操作は、CLI 自体よりもかなり遅いので、単一のファイルに対してのみ実行することをお勧めします。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:124]:
```suggestion
デーモン経由の操作は、CLI 経由よりもかなり遅いので、単一のファイルに対してのみ実行することをお勧めします。
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+:::
+
+### デーモンのログ
+
+Biome デーモンはファイルにログを保存します。ログは `biome-logs` というフォルダに保存されます。このフォルダは、OSの一時ディレクトリにあります。
```
Comment (nissy-dev) [inline website/src/content/docs/ja/guides/integrate-in-editor.mdx:129]:
```suggestion
Biome のデーモンは、ファイルシステムにログを保存します。ログは 、`biome-logs` というフォルダに保存され、このフォルダはOSの一時ディレクトリに作成されています。
```

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx (hunk 1 above)
Comment (nissy-dev) [reply website/src/content/docs/ja/guides/integrate-in-editor.mdx:65]: I seem the current link is no problem 👍

**Review by nissy-dev — CHANGES_REQUESTED** [review]
Thanks! I left some comments

File: website/src/content/docs/ja/guides/integrate-in-editor.mdx [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+
+
+Visual Studio Code マーケットプレイスページで、[Biome VS Code extension](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) をインストールしてください。
+
+Biome をデフォルトのフォーマッターにするには、[supported file](/ja/internals/language-support/) を開いて、以下の手順に従ってください:
+
+- **コマンドパレット**を開く（<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> または 表示 → コマンドパレット）
+- **ドキュメントのフォーマット...** を選択
+- **デフォルトのフォーマッターの設定...** を選択
+- Biomeを選択
```
[the remark this answers is not in the document]
Comment (Spice-Z) [reply website/src/content/docs/ja/guides/integrate-in-editor.mdx:25]: I found other pags uses the Japanese menu name.

https://github.com/biomejs/biome/blob/6739a5148b41e3d81b19722fffe17d97de4434dc/website/src/content/docs/ja/reference/vscode.mdx#L26-L29

Is it okay to use English menu name here ?

[the remark this answers is not in the document]
Comment (nissy-dev) [reply website/src/content/docs/ja/guides/integrate-in-editor.mdx:25]: I see, that's OK. Thank you for teaching me!

**Review by nissy-dev — APPROVED** [review]
Thank you for you contribution!

**Pull request merged**